public/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <title>F1ormula</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.0/css/bulma.min.css"/>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+
+  <section class="hero">
+    <div class="hero-body">
+      <div class="container brand">
+        <img id="f1ormula-brand" src="images/brand/f1ormula_brand_no-flag-01.png" alt="F1ormula" />
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="field">
+      <label class="label race-selectiontab">Select Round</label>
+      <div class="control race-selection">
+        <div class="select">
+          <select>
+            <option>Chinese Grand Prix</option>
+            <option>Bahrain Grand Prix</option>
+            <option>Russian Grand Prix</option>
+            <option>Gran Premio De Espana</option>
+            <option>Grand Prix De Monaco</option>
+            <option>Grand Prix Du Canada</option>
+            <option>Grosser Preis Von Osterreich</option>
+            <option>British Grand Prix</option>
+            <option>Magyar Nagydij Grand Prix</option>
+            <option>Belgian Grand Prix</option>
+            <option>Gran Premio D'Italia</option>
+            <option>Singapore Grand Prix</option>
+            <option>Malaysia Grand Prix</option>
+            <option>Japanese Grand Prix</option>
+            <option>United States Grand Prix</option>
+            <option>Gran Premio De Mexico</option>
+            <option>Gran Premio Do Brasil</option>
+            <option>Abu Dhabi Grand Prix</option>
+          </select>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <!-- <div class="circut column">
+  </div> -->
+<div id="title-bar">
+  <div class="columns">
+    <div class="column is-2"></div>
+    <div class= "qualify-title column is-4">
+      <div class= "qualify-title">
+        Qualifying Position</div>
+    </div>
+    <div class= " finish-title column is-4">
+      <div class="finish-title">
+        Finish Position</div>
+    </div>
+  <div class="column is-2"></div>
```

Comment:
Please review your indentation and be sure to indent children one level deep than their parent.